src/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,51 @@
+package coffee.order;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CoffeeOrderBoard {
+    private List<Order> orders;
+
+    public CoffeeOrderBoard() {
+        this.orders = new ArrayList<>();
+    }
+
+
+    public void add(Order order) {
+        if (order == null) {
+            throw new IllegalArgumentException("Order cannot be null");
+        }
+        order.setNumber(orders.size() + 1);
+        orders.add(order);
+    }
+
+    public Order deliver() {
+        if (orders.isEmpty()) {
+            throw new IllegalStateException("Queue is empty");
+        }
+        return orders.remove(0);
+    }
+
+    public Order deliver(int number) {
+        Order order = null;
+        for (int i = 0; i < orders.size(); i++) {
+            Order currentOrder = orders.get(i);
+            if (currentOrder.getNumber() == number) {
+                order = currentOrder;
+                orders.remove(i);
+                break;
+            }
+        }
+        if (order == null) {
+            throw new IllegalArgumentException("Order not found");
+        }
+        return order;
+    }
+
+    public void draw() {
+        System.out.println("============= \nNum | Name");

```

Comment:
Remember, I've told you not to use print statements? Make this method return a string with all information you need and return it. Then you could print it in `main`. Or use Logger otherwise